engines/e_afalg.c

```diff
@@ -124,27 +124,58 @@ static ossl_inline int io_read(aio_context_t ctx, long n, struct iocb **iocb)
     return syscall(__NR_io_submit, ctx, n, iocb);
 }
 
+/* A version of 'struct timespec' with 32-bit time_t
+   and suseconds_t.  */
```

Comment:
_su**b**seconds_, maybe?

Reply:
I just went with nanoseconds to make it clear.